airflow/models/baseoperator.py

```diff
@@ -714,12 +714,14 @@ class derived from this one results in the creation of a task object,
         execution, receiving a context dictionary; raising an exception will
         prevent the task from being executed.
 
-        |experimental|
+        |experimental| Note that passing a function to the pre_execute parameter is experimental,
+        while overriding .pre_execute in a custom operator is stable.
```

Comment:
```suggestion
        |experimental| Passing a function to this argument is experimental, although
        (re-)implementing ``pre_execute`` in a custom operator subclass is stable.
```

This is probably a little better? It might still be confusing to some users what *overriding* means. Not sure TBH.

Reply:
Thank you! Yes, I agree, implementing is probably the more common term :)